CHANGELOG.md

```diff
@@ -4,32 +4,53 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v0.16.0](https://github.com/voxpupuli/puppet-nginx/tree/v0.16.0) (2018-12-28)
+## [v0.16.0](https://github.com/voxpupuli/puppet-nginx/tree/v0.16.0) (2019-02-09)
```

Comment:
why not 1.0.0?

Reply:
he, I even have that on my slides :(
https://bastelfreak.de/cfgmgmtcamp2019/#32

I would love to include https://github.com/voxpupuli/puppet-nginx/pull/1155 in the 1.0.0 release.